test/clj/rems/api/test_applications.clj

```diff
@@ -1897,6 +1898,12 @@
                                                            :field/title {:en "text field"
                                                                          :fi "text field"
                                                                          :sv "text field"}
+                                                           :field/optional false}
+                                                          {:field/type :attachment
+                                                           :field/id "att"
+                                                           :field/title {:en "attachment"
+                                                                         :fi "attachment"
+                                                                         :sv "attachment"}
```

Comment:
At some point we should perhaps make all the localized values different in the tests.

Reply:
I fixed it in this test.